app/util/DragHandler.ts

```diff
@@ -99,8 +99,10 @@ export class DragHandler {
     this.centerSnapping = this.settingsService.views.values.General.CenterSnapping;
 
     // Load some attributes about the video canvas
-    this.baseWidth = this.videoService.baseWidth;
-    this.baseHeight = this.videoService.baseHeight;
+    const baseRes = this.videoService.baseResolutions[startEvent.display];
+    this.baseWidth = baseRes.baseWidth;
+    this.baseHeight = baseRes.baseHeight;
```

Comment:
i'm not sure these need to be instance variables anymore? looks like you aren't using them in `pageSpaceToCanvasSpace`

Reply:
Yes, they need to be there both because otherwise they won't be accessible to the class and because the draggable needs to know which display the item is on so that it can appropriate calculate when dragging